src/promptflow/promptflow/_sdk/_service/utils/utils.py

```diff
@@ -149,7 +147,19 @@ def is_pfs_service_healthy(pfs_port) -> bool:
         response = requests.get("http://localhost:{}/heartbeat".format(pfs_port))
         if response.status_code == 200:
             logger.debug(f"Promptflow service is already running on port {pfs_port}, {response.text}")
-            return True
+            match = re.search(r'"promptflow":"(.*?)"', response.text)
+            if match:
+                version = match.group(1)
+                is_healthy = version == get_promptflow_sdk_version()
```

Comment:
I would suggest use a new function. 

We mix the concept of is_healty and is_version_match here, at leatst we should update docstring or change this function name to reflect that.

Reply:
add docstring since healthy contain pfs is runnning and version match.